experimental/veggieseasons/lib/screens/details.dart

```diff
@@ -28,12 +28,11 @@ class ServingInfoChart extends StatelessWidget {
       builder: (context, snapshot) {
         final target = snapshot?.data ?? 2000;
         final percent = standardPercentage * 2000 ~/ target;
-        final themeData = CupertinoTheme.of(context);
 
         return Text(
           '$percent% DV',
+          style: CupertinoTheme.of(context).textTheme.textStyle,
```

Comment:
nit: is it needed to explicitly specify the text style? I'd expect the `Text` widget to inherit the style from the theme.

Reply:
Perhaps I've done something wrong in assigning the theme to the whole app, but removing the `style` makes the text black regardless of the IOS brightness.

Reply:
Does it work if you wrap the `DetailsScreen` widget in a `DefaultTextStyle(style: CupertinoTheme.of(context).textTheme.textStyle)` widget? Anyways not a big deal it just makes the code a little bit more verbose.